### PR TITLE
refactor: refactoring panic calls

### PR DESCRIPTION
**Pull request by robot9706**
depends on: #48

### CODE REVIEW

File: agent/pkg/dagent/dagent.go [hunk 1]
```diff
@@ -94,7 +95,7 @@ func Serve(cfg *config.Configuration) {
 		err := utils.ExecTraefik(context.TODO(), params, cfg)
 		if err != nil {
 			// we wanted to start traefik, but something is not ok, thus panic!
```
Comment (polaroi8d) [inline agent/pkg/dagent/dagent.go:97]: The log is communicative enough, I think we can remove this comment.

File: agent/pkg/dagent/utils/release.go [hunk 2]
```diff
@@ -48,14 +49,14 @@ func DraftRelease(instance string, versionData v1.VersionData, deployResponse v1
 
 	if err != nil {
 		log.Println(fmt.Sprintln("Version drafting error ", err))
+		return
 	}
```
Comment (polaroi8d) [inline agent/pkg/dagent/utils/release.go:53]:
```
content, err := yaml.Marshal(&release); err != nil {
		log.Println(fmt.Sprintln("Version drafting error ", err))
		return
	}
```

Refactor like this, what do you think? and @nandor-magyar

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: agent/pkg/dagent/utils/release.go (hunk 2 above)
Comment (robot9706) [reply agent/pkg/dagent/utils/release.go:53]: This makes it impossible to access `content` outside of the if scope.

**Comment by polaroi8d** [comment]
Minor comment from my side, btw LGTM (informal). Please @nandor-magyar check it too.

**Review by polaroi8d — APPROVED** [review]

**Review by nandor-magyar — APPROVED** [review]
Neat :sunglasses:

**Pull request merged**